src/main/java/magicbees/bees/BeeSpecies.java

```diff
@@ -186,6 +186,8 @@ public enum BeeSpecies {
             EnumTemperature.NORMAL, EnumHumidity.NORMAL, true, true),
     TC_CHAOS("TCChaos", "tenebrarum", BeeClassification.THAUMIC, 0xCCCCCC, BodyColours.THAUMCRAFT_SHARD,
             EnumTemperature.NORMAL, EnumHumidity.NORMAL, true, false),
+    TC_ESSENTIA("TCEssentia", "defaultium essentia apis", BeeClassification.THAUMIC, 0xCCCCCC,
```

Comment:
personally, I'd go with arcanus essentium, but hilarious either way.

Reply:
> personally, I'd go with arcanus essentium, but hilarious either way.

haha it was your choice

Reply:
Lmfaoooooo was it? Hilarious